flaml/autogen/agentchat/groupchat.py

```diff
@@ -12,6 +12,7 @@ class GroupChat:
     agents: List[Agent]
     messages: List[Dict]
     max_round: int = 10
+    admin_name: str = "Admin"  # the name of the admin agent
```

Comment:
Maybe check if admin_name exists here? So you don't need to throw expection in run_chat

Reply:
how do we check it "here"?

Reply:
My understand is `admin_name` must be the name of an agent in `agents`, Otherwise the `keyboardInterruption`  will be raised when human user tries to interrupt group chat? 

If my understand is correct it would be better to check if `admin_name` is legit(a.k.a is one of an `agents`) in constructor.

Reply:
`admin_name` is optional. There doesn't have to be an admin. Only when there is, we support using KeyboardInterruption to take over.